pyproject.toml

```diff
@@ -88,6 +88,10 @@ write_to = "arelle/_version.py"
 [tool.pytest.ini_options]
 xfail_strict = true
 empty_parameter_set_mark = "fail_at_collect"
+addopts = "--strict-markers"
+markers = [
+    "slow: marks tests as slow (deselect with '-m \"not slow\"')",
```

Comment:
Would you consider deselect being the `fast` flag instead or is that not something you control?

Reply:
That would require adding a mark to all tests except those that are slow. I copied this from [the pytest docs](https://docs.pytest.org/en/6.2.x/mark.html#registering-marks)